### PR TITLE
[#6718] Wallet onboarding setup re-style for new wallet flow

### LINKED ISSUE #6718: Wallet setup restyle for new wallet flow
Restyle the initial wallet screens so they resemble the screens for the new wallet flow 

Figma Wallet Setup
https://www.figma.com/file/SIeZPHvrX7KUwEBW8VQct8C5/WIP_New-send-transaction?node-id=0%3A1

<blockquote><div> Figma</div><div><strong><a href="https://www.figma.com/file/SIeZPHvrX7KUwEBW8VQct8C5/WIP_New-send-transaction">Figma</a></strong></div><div>Created with Figma</div></blockquote>

### PR DESCRIPTION
**Pull request by bhauman**
fixes #6718  Wallet onboarding setup re-style for new wallet flow

### Summary:

This re-styles the wallet onboarding screens according to for `wallet-01` in the following figma https://www.figma.com/file/SIeZPHvrX7KUwEBW8VQct8C5/WIP_New-send-transaction.

### Testing notes:

Check that the screens match the figma and that they function as intended.

### Steps to test:
- Open Status
- Create a new account
- Nav to wallet
- Click "Let's get setup"
- compare the screen to `wallet-01` on the figma linked above
- Click "Got it"
- compare the content of the alert dialog to the alert in `wallet-01`

Check the modal view as well:
- Ensure in Development Mode
- add the Status Test Dapp
- Open Status Test Dap
- Click "Transactions"
- Click "Send one Tx in batch"
- the setup modal should now be visible and should look correct

Check that once the signing phrase accepted that the onboarding screens don't show up when you return to the wallet.

Thanks!

status: ready

<blockquote><div> Figma</div><div><strong><a href="https://www.figma.com/file/SIeZPHvrX7KUwEBW8VQct8C5/WIP_New-send-transaction">Figma</a></strong></div><div>Created with Figma</div></blockquote>

### CODE REVIEW

File: src/status_im/ui/screens/wallet/onboarding/styles.cljs [hunk 1]
```diff
@@ -1,6 +1,68 @@
...
+
+(def signing-phrase
+  {:background-color colors/white
+   :border-radius    8
+   :flex-direction   :row})
+
+(def signing-word
+  {:flex            1
+   :height          66
+   :border-left-color "#ECECF0"
```
Comment (goranjovic) [inline src/status_im/ui/screens/wallet/onboarding/styles.cljs:18]: We should use the closest light gray already in colors namespace here

File: src/status_im/ui/screens/wallet/onboarding/styles.cljs [hunk 2]
```diff
@@ -1,6 +1,68 @@
...
+
+(def modal
+  {:flex             1
+   :background-color colors/blue})
+
+(def bottom-button-container
+  {:flex-direction :row,
+   :border-top-width 1
+   :background-color colors/blue
+   :border-top-color colors/white-light-transparent})
```
Comment (goranjovic) [inline src/status_im/ui/screens/wallet/onboarding/styles.cljs:44]: align map values in all maps

File: src/status_im/ui/screens/wallet/onboarding/styles.cljs [hunk 3]
```diff
@@ -40,4 +102,4 @@
    :margin-bottom    32})
 
 (def set-up-button-label
-  {:color "white"})
\ No newline at end of file
+  {:color "white"})
```
Comment (goranjovic) [inline src/status_im/ui/screens/wallet/onboarding/styles.cljs]: `colors/white`

**Review by goranjovic — APPROVED** [review]

File: src/status_im/ui/screens/wallet/onboarding/styles.cljs [hunk 4]
```diff
@@ -1,6 +1,68 @@
 (ns status-im.ui.screens.wallet.onboarding.styles
   (:require [status-im.ui.components.colors :as colors]))
 
+(def border-top-justify
+  {:justify-content :space-between
+   :border-top-color colors/white-light-transparent
+   :border-top-width 1
+   :flex 1})
```
Comment (dmitryn) [inline src/status_im/ui/screens/wallet/onboarding/styles.cljs]: Please indent/format map values

File: src/status_im/ui/screens/wallet/onboarding/views.cljs [hunk 5]
```diff
@@ -1,11 +1,118 @@
...
+  ;; want to move this somewhere for reuse
+  [react/view
+   {:style {:border-color "rgba(255,255,255,0.6)"
+            :border-width 1
+            :border-radius 8
+            :padding-top 15
+            :padding-bottom 15
+            :padding-left 20
+            :padding-right 20
+            :align-items :center}}
```
Comment (dmitryn) [inline src/status_im/ui/screens/wallet/onboarding/views.cljs]: I would move styles to `status-im.ui.screens.wallet.onboarding.styles` as i tend to not using inline styles unless it's really short. But this is point to be discussed, do we really need to set preference/rule for that in our code guidelines.

**Review by dmitryn — APPROVED** [review]
Some minor formatting issues, but looks good to me.

**Comment by asemiankevich** [comment]
UI mostly looks good for both devices. 

### iOS:

image

image

image



### Android:

image

@bhauman **please notice that info icon is a bit hidden on android ( Huawei P10 Lite, 5.2"), that should some kind of dynamic element or whatever.**

image

image

Bruce, could you please look into this UI issue meantime? I am continue testing now, just wanted to give you quick feedback. 

@denis-sharypin or @errorists please review the UI as well.

**Comment by asemiankevich** [comment]
@bhauman Issue i noticed just now: Send transaction screen (and Onboarding if not completed yet) is not shown for user, who tries to respond on request with an asset that is not enabled in his portfolio. Steps:

1. Recover account as user A. As user A do not complete onboarding. User A has ETH and STT shown in wallet assets by default
2. Recover account as user B. As user B complete onboarding. User B has ETH, STT and HND shown in wallet assets 
3. As user B request HND tokens in 1x1 chat with user A
4. As user A tap to respond the request and tap send

Actual: Onboarding screen is not shown
Expected: onboarding screen should be shown

@goranjovic maybe this was done intentionally and i missed that? Looks like bug to me now.

**Comment by bhauman** [comment]
@asemiankevich thanks so much, I will fix this.  I've been having trouble getting Android up and running. Sorry for sending this to you before really getting a good look at it on Android.

**Comment by bhauman** [comment]
@asemiankevich I believe the bug you found is separate a issue that exists in the current codebase.

**Comment by asemiankevich** [comment]
> @asemiankevich I believe the bug you found is separate a issue that exists in the current codebase.

yeah i believe so, checking it now in develop too (

**Comment by asemiankevich** [comment]
> @asemiankevich I believe the bug you found is separate a issue that exists in the current codebase.

i have added https://github.com/status-im/status-react/issues/6759 to our tracker. Maybe @goranjovic  will have some insight here :)

**Comment by asemiankevich** [comment]
@hetvart @yevh-berdnyk once the PR merged could you please make sure:

1 - this text on the screen is checked in e2e framework, because seems we dont care about the context of the screen, just tapping Got it button. However this screen has new copy that should be tracked.

image

2- this copy is updated (the confirmation pop-up)

image

**Comment by goranjovic** [comment]
@asemiankevich @bhauman the other issue is an existing one. We should figure out what to do here, as we don't really have it defined now.

@asemiankevich what happened to that info icon on the first screenshot of your last post, looks cut off.

**Comment by asemiankevich** [comment]
@goranjovic yeah Bruce said it will be fixed soon

**Review by errorists — COMMENTED** [review]
Looks great, the only thing which caught my attention is the 3-word combo text label on Android is using a dark grey color, it should be black.

**Comment by asemiankevich** [comment]
thanks @errorists ! @bhauman would you mind to pick this up too?

**Comment by bhauman** [comment]
@asemiankevich OK I've fixed the visual problems on Android.  What action am I supposed to take at this point?

I made the the signing words black and I made the (i) circle fully visible on Android.

**Comment by asemiankevich** [comment]
@bhauman would you mind to rebase your PR? we have some changes in e2e framework , that is the reason tests are failing.

**Comment by bhauman** [comment]
@asemiankevich I just rebased

**Comment by antdanchenko** [comment]
@bhauman I am going to add a commit to this PR in order to rework wallet setup flow in e2e tests, please mind those changes before pushing anything to this PR

**Comment by bhauman** [comment]
@antdanchenko you got it

**Comment by asemiankevich** [comment]
looks good
image

image

**Comment by antdanchenko** [comment]
@bhauman Can you add an `:accessibility-label` for this group, please? 
Something like `sign-in-phrase`?
Or just point me the group in comment and I will add the label by myself

As result we will be able to locate this 3 words in e2e tests
screen shot 2018-11-20 at 3 18 55 pm

**Comment by bhauman** [comment]
@antdanchenko 
https://github.com/status-im/status-react/pull/6748/files#diff-99d7df001cd21432078e50184c99e1edR69

<blockquote><div> GitHub</div><div><strong><a href="https://github.com/status-im/status-react/pull/6748">[#6718] Wallet onboarding setup re-style for new wallet flow by bhauman · Pull Request #6748 · status-im/status-react</a></strong></div><div>fixes #6718  Wallet onboarding setup re-style for new wallet flow
Summary:
This re-styles the wallet onboarding screens according to for wallet-01 in the following figma https://www.figma.com/file/...</div></blockquote>

**Comment by antdanchenko** [comment]
@bhauman please check my commit, if everything is fine with the label we can squash them in order to merge the PR

**Comment by bhauman** [comment]
Great I'll pull this in.

**Comment by antdanchenko** [comment]
@bhauman please wait a sec with merge, I did a mistake, need to add one more commit

**Comment by bhauman** [comment]
@antdanchenko So this is it or should I wait to rebase this?

**Comment by antdanchenko** [comment]
@bhauman test_wallet_set_up is fixed in https://github.com/status-im/status-react/pull/6748/commits/d4577f1dbc673fbfd6c82ee078a802691d15da93, nothing else to fix, everything is fine to be rebased and merged, sorry one more time

**Comment by bhauman** [comment]
@antdanchenko thanks just want to mention my other restyling PR that has wording changes https://github.com/status-im/status-react/pull/6797

**Pull request merged**